README.md

```diff
@@ -1,350 +1,61 @@
-# Beneficiary FHIR Data Server (BFD)
+Beneficiary FHIR Data Server (BFD)
 ====================================
 
 ## About
 
-Beneficiary FHIR Data (BFD) Server: The BFD Server is an internal backend system used at CMS to represent Medicare beneficiaries' demographic, enrollment, and claims data in [FHIR](https://www.hl7.org/fhir/overview.html) format.
+This project contains modules and documentation in support of the Beneficiary FHIR Data (BFD) Server. 
+The BFD Server is an API designed to serve Medicare beneficiaries' demographic, enrollment, and claims data in [FHIR](https://www.hl7.org/fhir/overview.html) format.
```

Comment:
While you're in here... indeed, it is a minor inconvenience, but the good people at Health Level Seven International would be pleased if we followed their [trademark guidance](https://www.hl7.org/fhir/license.html#trademark).

>  [..] This means that for any use of the trademarks ("FHIR" and the FHIR Icon) by parties other than HL7 and it's affiliates:
> [..]
> * When referencing the FHIR® standard in a website, document, presentation, or otherwise, please in a place of prominence refer to it as the "HL7® FHIR® standard". In subsequent uses, please refer to it as the "FHIR® standard" or "FHIR®", using the ® symbol as often as is practical, at least once on each page of printed matter, generally in connection with the first or dominant usage.

I sure hope GitHub doesn't have any issues rendering the US PTO trademark symbol "®" 🤞

```suggestion
The BFD Server is an API designed to serve Medicare beneficiaries' demographic, enrollment, and claims data using the [HL7® FHIR® Standard](https://www.hl7.org/fhir/overview.html) format.
```